examples/burgers1d.yml

```diff
@@ -67,12 +67,21 @@ latent_space:
   ae:
     hidden_units: [100]
     latent_dimension: 5
+    activation: softplus    
 
 latent_dynamics:
   type: sindy
   sindy:
     fd_type: sbp12
     coef_norm_order: fro
+    higher_order_terms: 1 
+    extra_functions: []
+  type: edmd
```

Comment:
This option either conflicts with or overwrites `type: sindy` at line 73. For beginners to start with, I'd commented out this line so that beginners start with sindy option first, but recognize there is an option for edmd?